prepare_tetragon_artifacts.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -e
+
+git_checkout_ref() {
+  local dir="$1"
+  local ref="$2"
+  git --work-tree="$dir" --git-dir="$dir/.git" checkout -q -f "$ref"
+  git --work-tree="$dir" --git-dir="$dir/.git" clean -xdfq
+}
+
+if [ $# -lt 1 ] ; then
+	echo "usage: $0 <TETRAGON_TAG>" 1>&2
+	exit 1
+fi
+
+TAG=$1
+VERSION="${TAG:1}"
+if [ -f "tetragon-${VERSION}.tgz" ]; then
+  echo "tetragon-${VERSION}.tgz already exists. Nothing to do."
+  exit 0
+fi
+
+echo "Generating tetragon package from tag: $TAG"
+rm -rf tetragon
+git clone git@github.com:cilium/tetragon.git
+git_checkout_ref "$(pwd)/tetragon" "$TAG"
+helm package -d . tetragon/install/kubernetes --version="${VERSION}" --app-version="${VERSION}"
+helm repo index . --merge index.yaml
+./generate_readme.sh > README.md
```

Comment:
Can we switch out the `$EDITOR README.md` on the `prepare_artifacts.sh` too? This was the one non-automated piece that I didn't get around to fixing :rocket:

Reply:
updated `prepare_artifacts.sh`. i also added `--no-edit` to the last `git commit --amend` command so that it doesn't open the editor.